src/constants.jl

```diff
@@ -67,6 +67,9 @@ const WORKTIME_BASE = 40
 
 const WEEK_DAYS_NO = 7
 const NUM_WORKING_DAYS = 5
+const SUNDAY_NO = 0
```

Comment:
Imo 7 lepsze niż 0.

Reply:
To wtedy `day_no -> day_no % WEEK_DAYS_NO != SUNDAY_NO` nie zadziała, chyba że zapisać to tam jako `day_no -> day_no % WEEK_DAYS_NO != (SUNDAY_NO % WEEK_DAYS_NO)`  co brzmi imho dziwniej niż `SUNDAY_NO = 0`.

Reply:
sorry żle popatrzyłem, oczywiście masz rację! mergujemy